airflow/settings.py

```diff
@@ -173,8 +173,8 @@ def configure_orm(disable_connection_pool=False):
         # https://docs.sqlalchemy.org/en/13/core/pooling.html#disconnect-handling-pessimistic
         pool_pre_ping = conf.getboolean('core', 'SQL_ALCHEMY_POOL_PRE_PING', fallback=True)
 
-        log.info("settings.configure_orm(): Using pool settings. pool_size={}, max_overflow={}, "
-                 "pool_recycle={}, pid={}".format(pool_size, max_overflow, pool_recycle, os.getpid()))
+        log.debug("settings.configure_orm(): Using pool settings. pool_size=%d, max_overflow=%d, "
+                  "pool_recycle=%d, pid=%d", pool_size, max_overflow, pool_recycle, os.getpid())
```

Comment:
I think this should be a separate PR. This log will not only be displayed when printing airflow version.

Reply:
You are right, however, I am not sure if this log information is so crucial that it deserves separate PR? @feluelle @mik-laj

Reply:
I think it ok, just a small change, unless we create a PR change all log and and some static test for it